engines/bops_core/app/components/bops_core/status_detail_component.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+module BopsCore
+  class StatusDetailComponent < GovukComponent::Base
+    renders_one :title
+    renders_one :body
+    renders_one :status
+
+    attr_reader :id, :open
+
+    def initialize(id: nil, open: false, classes: [], html_attributes: {})
+      @id = id
+      @open = open
+      super(classes:, html_attributes:)
+    end
+
+    def before_render
+      fail_missing_attributes
+    end
+
+    def call
+      tag.div(**html_attributes) do
+        safe_join([details_block, status_block].compact_blank)
```

Comment:
Not sure the `compact_blank` is necessary here because wouldn't it be an error for either the body or the status not to be present?

Reply:
Added the validation after and forgot to take this out, thanks